.github/workflows/on-workflow-call-blog-container-deployment.yml

```
@@ -1,19 +1,22 @@
-# this is deactivated
-name: CI
+name: CD / Blog to Koyeb
 
 on:
   workflow_call:
+    inputs:
+      NODE_ENV:
+        required: true
+        default: "production"
```

actionlint (1.7.4)

input "NODE_ENV" of workflow_call event has the default value "production", but it is also required. if an input is marked as required, its default value will never be used

(events)